resources/lang/dk_DA.json

```
@@ -0,0 +1,33 @@
+{
+   "noRecordsFound":         "Ingen felter fundet",
+   "showingPage":            "Viser side {0} af {1}",
+   "show":                   "Vis ",
+   "search":                 "Søg",
+   "entries":                "indgange",

```

Translating "entries" to "indtastninger" would probably be more accurate.

```suggestion
   "entries":                "indtastninger",

```